libtenzir/src/application.cpp

```diff
@@ -83,6 +83,10 @@ void add_root_opts(command& cmd) {
     "plugins to load at startup; the special values 'bundled' "
     "and 'all' enable autoloading of bundled and all plugins "
     "respectively.");
+  cmd.options.add<caf::config_value::list>(
+    "?tenzir", "disabled-plugins",
```

Comment:
@tobim @jachris before I write documentation for this, let's agree on the name for this option.

Reply:
`disable-plugins` would fit better with other option names, but I don't care too much TBH.

Reply:
I changed it accordingly.